recommendations/graduated/harbor.md

```diff
@@ -0,0 +1,75 @@
+# CNCF SIG Runtime Graduation Review Project Request
+
+## Harbor Graduation Proposal
+
+Since joining the CNCF in [July 2018](https://www.cncf.io/blog/2018/07/31/cncf-to-host-harbor-in-the-sandbox/) as a Sandbox project and in [November 2018](https://www.cncf.io/blog/2018/11/13/harbor-into-incubator/) moving to an Incubating Project, Harbor has built a healthy ecosystem of users, maintainers, and production implementations.
+
+On behalf of the maintainers team, we believe Harbor is ready for the CNCF [Graduation stage](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#graduation-stage) and meets the v1.2 criteria. We address the CNCF Technical Due Diligence questions in a separate Google Docs document (referred to as the `Tech-DD` from now on).
+
+*Name of project:* Harbor
+
+*Project Description:* Harbor is an open source cloud native registry that provides trust, compliance, performance, and interoperability. As a private on-premises registry, Harbor fills a gap for organizations that prefer not to use a public or cloud-based registry or want a consistent experience across clouds. Harbor secures images with role-based access control, scans images for vulnerabilities, and signs images as trusted. A CNCF Incubating project, Harbor helps you consistently and securely manage images across cloud native compute platforms like Kubernetes and Docker.
+
+Harbor in essense solves common problems in organizations building cloud native applications by offering them a choice to deploy their own private on-premises registry and apply a consistent policy across artifact management in any cloud
+
+Harbor is currently a CNCF Incubating Project. 
+
+## Collateral
+
+**_Link to TOC PR_**
+
+https://github.com/cncf/toc/pull/311
+
+**_Link to Presentation Slides_**
+
+https://docs.google.com/presentation/d/12B5dGc8lMxKeKBFS8pGoldBtZaIWBtDYUFSNcG0mnCc/edit?ts=5d9f4ab2#slide=id.g5d0c4feb44_0_0
+
+**_Link to TOC Presentation_**
+
+https://youtu.be/VRo0U36CZFU?list=PLj6h78yzYM2Mf6GCZzW6CAk6GlZESbemB&t=1785
+
+**_Link to GitHub project_**
+
+https://github.com/goharbor
+
+**_Link to Tech-DD_**
+
+https://docs.google.com/document/d/15gX7EeeXQThEvVMGpL-0a1mOwGuByLtMfvXNJaKT0A0/edit?usp=sharing
+
+**_Link to SIG-Runtime Tech-DD for Harbor Graduation_**
+
+http://bit.ly/harbor-graduation-dd
```

Comment:
this doc is still WIP?

Reply:
hi @k28cn, this is the PR that will do the work that goes into that DD doc (essentially verifying compliance with CNCF graduation criteria).

Reply:
Great! I'll share my points after went through the related doc :)